ios/Classes/SwiftWootricsdkFlutterPlugin.swift

```diff
@@ -102,6 +102,50 @@ public class SwiftWootricsdkFlutterPlugin: NSObject, FlutterPlugin {
                     Wootric.setCustomLanguage(languageCode)
                 }
 
+            case "setSliderColor":
+                if let arguments = call.arguments as? [String: Any] {
+                    let hexColor = arguments["color"] as? String
+                    guard hexColor != nil else {
+                      return
+                    }
+                    if let color = convertHEXtoUIColor(hexColor!) {
+                      Wootric.setSliderColor(color)
+                    }
+                }
+
```

Comment:
Above code can be optimise as below,

```
if let arguments = call.arguments as? [String: Any],
   let hexColor = arguments["color"] as? String {
         if let color = convertHEXtoUIColor(hexColor) {
                  Wootric.setSliderColor(color)
            }
          }
```

Similar code can be used for other cases as well.